proto/RobotCommands.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package proto;
+
+message RobotCommand {
+    uint32 id = 1;
+
+    float velocity_x = 2;
+    float velocity_y = 3;
+    float angle = 4;
+    float angular_velocity = 5;
+    bool use_angular_velocity = 6;
+
+    float camera_angle_of_robot = 7;
+    bool camera_angle_of_robot_is_set = 8;
```

Comment:
Same as above, you don't need the boolean; just use proto's has_camera_angle_of_robot()

Reply:
Same here as the other comment